### PR TITLE
Added Profile page

### PR DESCRIPTION
**Pull request by saurabhchirde**
Added a simple profile page just to show user details with the total note counts for all different categories.

### CODE REVIEW

**Review by j24m — APPROVED** [review]
Code is clean , and it looks fine to me .

File: src/Pages/ProfilePage/ProfilePage.jsx [hunk 1]
```diff
@@ -1,5 +1,70 @@
...
+  } = useAuth();
+  const {
+    state: { deletedNotes, allNotes, archivedNotes },
+  } = usePebbleNote();
+  const { setError, setShowError } = useModal();
+  const navigate = useNavigate();
+
+  const logOutHandler = () => {
+    authDispatch({ type: "logout" });
+    setError("Your have been logged out");
```
Comment (muskanbagrecha) [inline src/Pages/ProfilePage/ProfilePage.jsx:21]: It is somewhat ambiguous that you are treating the case of logout as an error due to the variable names.

**Review by muskanbagrecha — APPROVED** [review]
Small PR, looks good. But please fix this
image

File: src/Pages/ProfilePage/ProfilePage.jsx (hunk 1 above)
Comment (saurabhchirde) [reply src/Pages/ProfilePage/ProfilePage.jsx:21]: The variable was created long ago, I will fix this.

**Pull request merged**